packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -40,7 +40,9 @@ describe('Approve token for deposit', () => {
       cy.findByRole('button', {
         name: /Pay approval fee of/
       }).click()
-      cy.confirmMetamaskPermissionToSpend('1')
+      cy.confirmSpending('5')
+      cy.wait(10_000)
+      cy.rejectMetamaskTransaction()
```

Comment:
Why do we want to reject the transaction? We can assert by confirming the next metamask transaction too, right? That way we wouldn't need to wait and it'd be cleaner too!

Reply:
We need to wait whether we reject or confirm

Reply:
I understand the wait, but why are we rejecting the tx +1?

Reply:
Rejecting is more likely to succeed than confirming. I don't mind accepting it though